lib/src/presentation/view/movie_details.dart

```diff
@@ -23,6 +27,9 @@ class _MovieDetailsState extends State<MovieDetails> {
   static const String originalTitle = 'Original title:';
   static const String overviewTitle = 'Overview';
   static const String genreTitle = "Genres";
+  static const String genreTitleKey = 'genre title';
+  static const String floatingActionButtonKey = 'counter_button';
```

Comment:
If you want you can move all the keys that you used on the widget to a specific file for WidgetKeys